apps/cowswap-frontend/src/api/gnosisProtocol/errors/QuoteError.ts

```diff
@@ -55,6 +57,13 @@ export function mapOperatorErrorToQuoteError(error?: ApiErrorObject): GpQuoteErr
         errorType: GpQuoteErrorCodes.TransferEthToContract,
         description: error.description,
       }
+
+    case ApiErrorCodes.SameBuyAndSellToken:
+      return {
+        errorType: GpQuoteErrorCodes.SameBuyAndSellToken,
+        description: GpQuoteErrorDetails.SameBuyAndSellToken,
+      }
```

Comment:
You need to add a mapping to `apps/cowswap-frontend/src/modules/tradeFormValidation/pure/TradeFormButtons/tradeButtonsMap.tsx`

Build is failing https://vercel.com/cowswap/swap-dev/u6vkfG9h4FMHGhD1j4fjanfgSUVK

![image](https://github.com/cowprotocol/cowswap/assets/43217/8f51e86e-0667-481d-92ab-34f1bcc08290)

Reply:
Fixed! thanks

Reply:
it builds now :)